React/Base/RCTUtils.m

```diff
@@ -1067,3 +1067,8 @@ RCT_EXTERN BOOL RCTValidateTypeOfViewCommandArgument(
 
   return true;
 }
+
+BOOL RCTIsAppActive(void)
+{
+  return [RCTSharedApplication() applicationState] == UIApplicationStateActive;
+}
```

Comment:
Sorry to bother, but our linter is asking a new line at the end of the file. Could you please add it?

Reply:
done, and pushed